site/src/docs/getting-started/versioning.mdx

```diff
@@ -0,0 +1,60 @@
+---
+slug: '/getting-started/versioning'
+title: 'Versioning'
+---
+
+import ExternalLink from '../../components/ExternalLink';
+import LeadParagraph from '../../components/LeadParagraph';
+import InternalLink from '../../components/InternalLink';
+
+# Versioning in HDS
+
+<LeadParagraph>
+  HDS follows <ExternalLink href="https://semver.org/">Semver versioning</ExternalLink>. This page describes HDS
+  practices which versions are supported and what is considered as breaking changes.
+</LeadParagraph>
+
+## Which versions are supported?
+
+HDS supports current and one previous major version. Older versions are not supported anymore. We recommend updating to the latest version.
+
+## What is considered as a breaking change?
+
+Breaking change can occur either in design or in the implementation or in both.
+
+### Design
+
+These are breaking changes
+
+- if released component is removed completely
+- width of the released component changes (causing other elements possibly flowing to new row)
+- typography changes (e.g. font-size, font weight, letter-spacing ) on already existing styles
+- changing spacing outside the element (margins)
+- overridable elements are changed/removed (this causes overrides to be reset in component instances)
+- color token values are changed
+
+These are not breaking changes
+
+- component structure is updated (e.g. layer order) but nothing is removed
+- changing padding inside the element, so that the outside stays the same size
+- symbol name or symbol hierarchy is changed
+- layer names are changed
+- border radius is changed
+- (default) color is changed inside the component
+- elevation of the component is changed
+
+### Implementation
+
+These are breaking changes
+
+- properties are removed completely
+- renaming existing components or properties
+- changes in component box model affecting how the components align with each other or the width of the element
+- if it is a breaking change in design, it most likely is also in the implementation
+- updating dependency so that previous versions are not supported anymore
```

Comment:
I'm not sure what this means 🤔 
Is it React update for the library or something like that? Or is it dropping legacy, aka deprecated components?
Should it have a mention of deprecated?